deploy/kubernetes/webhook-example/webhook.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: snapshot-validation-deployment
+  namespace: default # NOTE: change the namespace
+  labels:
+    app: snapshot-validation
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: snapshot-validation
+  template:
+    metadata:
+      labels:
+        app: snapshot-validation
+    spec:
+      containers:
+      - name: snapshot-validation
+        image: gcr.io/your-project-name/validation-webhook:latest # change the image if you wish to use your own custom validation server image
```

Comment:
Since we are releasing this together with snapshot-controller and csi-snapshotter, the version will be v2.2.0.
```
k8s.gcr.io/sig-storage/validation-webhook:v2.2.0
```